data/language/en-GB.txt

```diff
@@ -4199,6 +4199,8 @@ STR_5887    :Prefix
 STR_5888    :Suffix
 STR_5889    :Custom currency symbol
 STR_5890    :Enter the currency symbol to display
+STR_5898    :Default
```

Comment:
just for the record i have stopped adding strings at 5899 and will likely not add any more. also, don't feel you need to skip any of the strings due to my pull requests, as if yours gets merged before mine, i'll merely update the string ids in my PR ^_^. 

I have a feeling that the devs don't want us to skip strings for other pull requests anyway (at least, this is what i am merely assuming, i'm not entirely sure on this), as the other pull requests aren't always guaranteed to be merged.